compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Dialog.desktop.kt

```diff
@@ -272,11 +272,15 @@ fun Dialog(
             it.compositionLocalContext = compositionLocalContext
             it.exceptionHandler = windowExceptionHandlerFactory.exceptionHandler(it)
 
+            val wasDisplayable = it.isDisplayable
+
             update(it)
 
-            if (!it.isDisplayable) {
-                it.makeDisplayable()
```

Comment:
Removing this breaks the feature `draw the content before the first isVisible=true` with a specified window size (we don't do `pack` in this case)

Reply:
Noticied the changes in `setSizeSafely`. But `setSizeSafely` theoretically can be not called. In a case when the window is already this size. So, we won't call `pack`.

Reply:
Fixed.

Reply:
> setSizeSafely must be called at least once, because the check is

I see, but will it always be the case?

There is one case I noticied, we call it only if

> placement == WindowPlacement.Floating

(search for it in this file)

Also, is it okay to call `pack` every time when we specify the size?

Reply:
Take a look at the latest version.